src/package/package.rs

```diff
@@ -100,6 +100,10 @@ impl Package {
         }
     }
 
+    pub fn display_name_version(self) -> String {
+        format!("{}-{}", self.name, self.version)
```

Comment:
I'd personally prefer a "{name} {version}" as format as the version might contain dashes but shouldn't contain spaces - what do you think?